python/setup.py

```diff
@@ -369,20 +459,40 @@ def build(build_python, build_java, build_cpp):
         logger.warning("Expected Bazel version {} but found {}".format(
             ".".join(map(str, SUPPORTED_BAZEL)), bazel_version_str))
 
+    bazel_flags = ["--verbose_failures"]
+
+    if not is_automated_build:
+        bazel_precmd_flags = []
+    if is_automated_build:
+        root_dir = os.path.join(
+            os.path.abspath(os.environ["SRC_DIR"]), "..", "bazel-root")
```

Comment:
Hi @krfricke , Why we don't set `ROOT_DIR` directly instead of setting `SRC_DIR` and then cancating 
 to `SRC_DIR/../`?

Reply:
`SRC_DIR` is set by [conda build](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-set-during-the-build-process), and `ROOT_DIR` is not set by anything AFAIK. Yes, I'm a 🐌